javascript/ql/src/semmle/javascript/frameworks/Electron.qll

```diff
@@ -16,7 +16,7 @@ module Electron {
   /**
    * An instantiation of `BrowserWindow` or `BrowserView`.
    */
-  abstract private class NewBrowserObject extends BrowserObject {
+  abstract private class NewBrowserObject extends BrowserObject, DataFlow::TrackedNode {
```

Comment:
Could do, but we'd have to see whether that still scales since there could be a lot more `BrowserObjectByType`s than `NewBrowserObject`s. (In a way, with `BrowserObjectByType` we let the TypeScript type inference do the tracking for us.)